cfgov/unprocessed/js/apps/owning-a-home/ExpandableGroup.js

```diff
@@ -0,0 +1,67 @@
+// Required modules.
+import { checkDom as _checkDom, setInitFlag as _setInitFlag }
```

Comment:
Underscore prefix is for designating a private function within a module. It seems kinda weird to be using it here when the function is coming into the module publicly.

Reply:
I guess it's okay if this is temp code.